server/src/db/migrations/20210126102343_createCategories.cjs

```diff
@@ -17,6 +17,6 @@ exports.up = async (knex) => {
 /**
  * @param {Knex} knex
  */
-exports.down = (knex) => {
+exports.down = async (knex) => {
```

Comment:
The reason the migration did not include `async` is because it's not included in the migration stub. You should update the `migration.stub.cjs` so that everytime you generate a new migration the `down` will include `async`

Make sure everyone is aware on your team that you modified an existing migration file